### PR TITLE
Merge latest NationalMap#master, upgrade to TerriaJS 7.2.1

### PR DESCRIPTION
**Pull request by meh9**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,9 +25,6 @@ import render from './lib/Views/render';
 import GnafAddressGeocoder from 'terriajs/lib/Map/GnafAddressGeocoder.js';
 import ViewerMode from 'terriajs/lib/Models/ViewerMode.js';
 
-// Tell the OGR catalog item where to find its conversion service.  If you're not using OgrCatalogItem you can remove this.
-OgrCatalogItem.conversionServiceBaseUrl = configuration.conversionServiceBaseUrl;
```
Comment (meh9) [inline index.js]: Oops, we should probably have kept that.

File: index.js [hunk 2]
```diff
@@ -34,7 +26,7 @@ import render from './lib/Views/render';
 import GnafAddressGeocoder from 'terriajs/lib/Map/GnafAddressGeocoder.js';
 import ViewerMode from 'terriajs/lib/Models/ViewerMode.js';
 
-// Tell the OGR catalog item where to find its conversion service.  If you're not using OgrCatalogItem you can remove this.
+// Tell the OGR catalog item where to find its conversion service.
 OgrCatalogItem.conversionServiceBaseUrl = configuration.conversionServiceBaseUrl;
```
Comment (reginapramesti) [inline index.js]: This line is giving an error because `configuration` is undefined

File: index.js [hunk 3]
```diff
@@ -5,14 +5,6 @@
...
     baseUrl: 'build/TerriaJS'
 };
-var configuration = {
-    bingMapsKey: undefined, // use Cesium key
-};
-
-// Check browser compatibility early on.
-// A very old browser (e.g. Internet Explorer 8) will fail on requiring-in many of the modules below.
-// 'ui' is the name of the DOM element that should contain the error popup if the browser is not compatible
-//var checkBrowserCompatibility = require('terriajs/lib/ViewModels/checkBrowserCompatibility');
```
Comment (reginapramesti) [inline index.js]:
```suggestion
var configuration = {
    bingMapsKey: undefined, // use Cesium key
};
```

File: index.js (hunk 2 above)
Comment (meh9) [reply index.js]: Turns out we don't need the OGR thing here any more so I've removed the reference.

File: index.js (hunk 3 above)
Comment (meh9) [reply index.js]: Don't need this any more.

**Pull request merged**